src/service/request-service.js

```diff
@@ -85,20 +85,31 @@ class RequestService {
   }
 
   /**
-   * Sets "Was split?" to "yes" in Airtable
+   * Sets "Task" column to the first task
    *
-   * @param {object} request Original request record to be marked as split
-   * @returns {void}
+   * @param {object} request Original request record
+   * @returns {Array} The remaining tasks, after the first is removed
    */
-  markRequestAsSplit(request) {
-    this.base.update(request.id, { "Was split?": "yes" }, (err) => {
+  useFirstRequestTask(request) {
+    const { tasks } = request;
+    this.base.update(request.id, { Tasks: [tasks.shift().rawTask] }, (err) => {
```

Comment:
My suggestion would be to not modify the `tasks` array here (`shift()` is a destructive) operation. Instead, update the column and create new rows with subsequent tasks. Then let the bot pick up the updated row in the next poll.